proposals/kubeflow-steering-committee-election-proposal.md

```diff
@@ -0,0 +1,125 @@
+Interim KSC member B</td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>2024</td>
+      <td>New election</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>2025</td>
+      <td></td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>2026</td>
+      <td>New election</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>2027</td>
+      <td></td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>…</td>
+      <td></td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+## Election Procedure
+
+### Timeline ###  
+Steering Committee elections are held annually. 4 weeks or more before the election, the Steering Committee will appoint Election Officer(s) (see below). 2 weeks or more before the election, the Election Officer(s) will issue a call for nominations, publish the list of eligible voters, and open the call for exceptions. One week before the election the call for nominations and exceptions will be closed. The election will be open for voting not less than two weeks and not more than four. The results of the election will be announced within one week of closing the election. New Steering Committee members will take office in December of each year on the date the results are announced.
+
+### Election Officer(s) ###  
+4 weeks or more before the election, the Steering Committee will appoint between one and three Election Officer(s) to administer the election. Elections Officers will be Kubeflow community members in good standing who are eligible to vote, are not running for Steering in that election, who are not currently part of the Steering Committee and can make a public promise of impartiality. They will be responsible for:
+
+-  Making all announcements associated with the election
+-  Preparing and distributing electronic ballots
+-  Judging exception requests
+-  Assisting candidates in preparing and sharing statements
+-  Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote ###
+Anyone who has at least 50 contributions in the last 12 months is eligible to vote in the Steering election. Contributions are defined as opening PRs, reviewing and commenting on PRs, opening and commenting on issues, writing design docs, commenting on design docs, helping people on slack, participating in working groups, and other efforts that help advance the Kubeflow project.
+
+This dashboard **\<TODO\>** shows only GitHub based contributions and does not capture all the contributions we value. We expect this metric not to capture everyone who should be eligible to vote. If a community member has had significant contributions over the past year but is not captured in the dashboard, they will be able to submit an exception form to the Elections Officer(s) who will then review and determine whether this member should be eligible to vote. All exceptions, and the reasons for them, will be recorded in a log that will be available to Steering and the TOC.
```

Comment:
Before adopting this proposal we MUST replace this TODO.

Reply:
Done.

Reply:
I believe that eleko requires a voters.yaml file.   https://elekto.dev/docs/administration/creating/#the-votersyaml-file.   Perhaps after the 1st version, it become easier as not that many will be add/deleted.   I would like to put the responsibility on the vote to add his/her name to the voters.yaml file and perhaps have the election official approve.